{
  "uid" : "6ea56f9ca8880d45",
  "name" : "Change the password by receiving an email link",
  "fullName" : "tests.profile_page_test.TestProfilePage#test_change_password",
  "historyId" : "9957e716460bffae2f2d96bf28fa1581",
  "time" : {
    "start" : 1728608229607,
    "stop" : 1728608229607,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5618cc43602a <unknown>\n#1 0x5618cc11c5e0 <unknown>\n#2 0x5618cc1147eb <unknown>\n#3 0x5618cc104a09 <unknown>\n#4 0x5618cc106744 <unknown>\n#5 0x5618cc104cbd <unknown>\n#6 0x5618cc10452f <unknown>\n#7 0x5618cc104422 <unknown>\n#8 0x5618cc102493 <unknown>\n#9 0x5618cc102ada <unknown>\n#10 0x5618cc11ef41 <unknown>\n#11 0x5618cc1b08c5 <unknown>\n#12 0x5618cc190b22 <unknown>\n#13 0x5618cc1afd7d <unknown>\n#14 0x5618cc1908c3 <unknown>\n#15 0x5618cc15e6b3 <unknown>\n#16 0x5618cc15f68e <unknown>\n#17 0x5618cc400a2b <unknown>\n#18 0x5618cc4049b1 <unknown>\n#19 0x5618cc3ed225 <unknown>\n#20 0x5618cc405532 <unknown>\n#21 0x5618cc3d238f <unknown>\n#22 0x5618cc424f28 <unknown>\n#23 0x5618cc4250f3 <unknown>\n#24 0x5618cc434e7c <unknown>\n#25 0x7ff54d294ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"6997b541d6184c95f009ef9997dc015c\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fb25e17e5a0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5618cc4250f3 \\\\u003Cunknown>\\\\n#24 0x5618cc434e7c \\\\u003Cunknown>\\\\n#25 0x7ff54d294ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5618cc43602a <unknown>\nE       #1 0x5618cc11c5e0 <unknown>\nE       #2 0x5618cc1147eb <unknown>\nE       #3 0x5618cc104a09 <unknown>\nE       #4 0x5618cc106744 <unknown>\nE       #5 0x5618cc104cbd <unknown>\nE       #6 0x5618cc10452f <unknown>\nE       #7 0x5618cc104422 <unknown>\nE       #8 0x5618cc102493 <unknown>\nE       #9 0x5618cc102ada <unknown>\nE       #10 0x5618cc11ef41 <unknown>\nE       #11 0x5618cc1b08c5 <unknown>\nE       #12 0x5618cc190b22 <unknown>\nE       #13 0x5618cc1afd7d <unknown>\nE       #14 0x5618cc1908c3 <unknown>\nE       #15 0x5618cc15e6b3 <unknown>\nE       #16 0x5618cc15f68e <unknown>\nE       #17 0x5618cc400a2b <unknown>\nE       #18 0x5618cc4049b1 <unknown>\nE       #19 0x5618cc3ed225 <unknown>\nE       #20 0x5618cc405532 <unknown>\nE       #21 0x5618cc3d238f <unknown>\nE       #22 0x5618cc424f28 <unknown>\nE       #23 0x5618cc4250f3 <unknown>\nE       #24 0x5618cc434e7c <unknown>\nE       #25 0x7ff54d294ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728608229608,
      "stop" : 1728608230375,
      "duration" : 767
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728608130626,
      "stop" : 1728608130626,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728608230375,
      "stop" : 1728608230849,
      "duration" : 474
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5618cc43602a <unknown>\n#1 0x5618cc11c5e0 <unknown>\n#2 0x5618cc1147eb <unknown>\n#3 0x5618cc104a09 <unknown>\n#4 0x5618cc106744 <unknown>\n#5 0x5618cc104cbd <unknown>\n#6 0x5618cc10452f <unknown>\n#7 0x5618cc104422 <unknown>\n#8 0x5618cc102493 <unknown>\n#9 0x5618cc102ada <unknown>\n#10 0x5618cc11ef41 <unknown>\n#11 0x5618cc1b08c5 <unknown>\n#12 0x5618cc190b22 <unknown>\n#13 0x5618cc1afd7d <unknown>\n#14 0x5618cc1908c3 <unknown>\n#15 0x5618cc15e6b3 <unknown>\n#16 0x5618cc15f68e <unknown>\n#17 0x5618cc400a2b <unknown>\n#18 0x5618cc4049b1 <unknown>\n#19 0x5618cc3ed225 <unknown>\n#20 0x5618cc405532 <unknown>\n#21 0x5618cc3d238f <unknown>\n#22 0x5618cc424f28 <unknown>\n#23 0x5618cc4250f3 <unknown>\n#24 0x5618cc434e7c <unknown>\n#25 0x7ff54d294ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728608230375,
        "stop" : 1728608230849,
        "duration" : 474
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5618cc43602a <unknown>\n#1 0x5618cc11c5e0 <unknown>\n#2 0x5618cc1147eb <unknown>\n#3 0x5618cc104a09 <unknown>\n#4 0x5618cc106744 <unknown>\n#5 0x5618cc104cbd <unknown>\n#6 0x5618cc10452f <unknown>\n#7 0x5618cc104422 <unknown>\n#8 0x5618cc102493 <unknown>\n#9 0x5618cc102ada <unknown>\n#10 0x5618cc11ef41 <unknown>\n#11 0x5618cc1b08c5 <unknown>\n#12 0x5618cc190b22 <unknown>\n#13 0x5618cc1afd7d <unknown>\n#14 0x5618cc1908c3 <unknown>\n#15 0x5618cc15e6b3 <unknown>\n#16 0x5618cc15f68e <unknown>\n#17 0x5618cc400a2b <unknown>\n#18 0x5618cc4049b1 <unknown>\n#19 0x5618cc3ed225 <unknown>\n#20 0x5618cc405532 <unknown>\n#21 0x5618cc3d238f <unknown>\n#22 0x5618cc424f28 <unknown>\n#23 0x5618cc4250f3 <unknown>\n#24 0x5618cc434e7c <unknown>\n#25 0x7ff54d294ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"6997b541d6184c95f009ef9997dc015c\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728608230880,
      "stop" : 1728608230942,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Profile Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "profile_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestProfilePage"
  }, {
    "name" : "host",
    "value" : "fv-az523-260"
  }, {
    "name" : "thread",
    "value" : "2358-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.profile_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 27,
        "skipped" : 0,
        "passed" : 470,
        "unknown" : 0,
        "total" : 497
      },
      "items" : [ {
        "uid" : "3ab4a226b93b7136",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/3ab4a226b93b7136",
        "status" : "passed",
        "time" : {
          "start" : 1728557218162,
          "stop" : 1728557232422,
          "duration" : 14260
        }
      }, {
        "uid" : "b9952f77db2f9906",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/b9952f77db2f9906",
        "status" : "passed",
        "time" : {
          "start" : 1728554104090,
          "stop" : 1728554115935,
          "duration" : 11845
        }
      }, {
        "uid" : "19e2661074ef2180",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/19e2661074ef2180",
        "status" : "passed",
        "time" : {
          "start" : 1728522088829,
          "stop" : 1728522103215,
          "duration" : 14386
        }
      }, {
        "uid" : "5dc4ad81fc7b6f72",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/5dc4ad81fc7b6f72",
        "status" : "passed",
        "time" : {
          "start" : 1728470513131,
          "stop" : 1728470525914,
          "duration" : 12783
        }
      }, {
        "uid" : "e3f3bbe6e3450aac",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/e3f3bbe6e3450aac",
        "status" : "passed",
        "time" : {
          "start" : 1728467932160,
          "stop" : 1728467944932,
          "duration" : 12772
        }
      }, {
        "uid" : "90b70ad95bdf897b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/90b70ad95bdf897b",
        "status" : "passed",
        "time" : {
          "start" : 1728435807107,
          "stop" : 1728435821498,
          "duration" : 14391
        }
      }, {
        "uid" : "c1001bc242f851ba",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/c1001bc242f851ba",
        "status" : "passed",
        "time" : {
          "start" : 1728378030610,
          "stop" : 1728378042844,
          "duration" : 12234
        }
      }, {
        "uid" : "d4af9d834bfe1c18",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/d4af9d834bfe1c18",
        "status" : "passed",
        "time" : {
          "start" : 1728375237118,
          "stop" : 1728375249353,
          "duration" : 12235
        }
      }, {
        "uid" : "c6877213c903894c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/c6877213c903894c",
        "status" : "passed",
        "time" : {
          "start" : 1728375121895,
          "stop" : 1728375133395,
          "duration" : 11500
        }
      }, {
        "uid" : "27d6146137823f22",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/27d6146137823f22",
        "status" : "passed",
        "time" : {
          "start" : 1728349205589,
          "stop" : 1728349218223,
          "duration" : 12634
        }
      }, {
        "uid" : "2007fd16c97660ba",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/2007fd16c97660ba",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('css selector', 'h3')\nStacktrace:\n#0 0x55b24eedb13a <unknown>\n#1 0x55b24ebc15e0 <unknown>\n#2 0x55b24ec10be8 <unknown>\n#3 0x55b24ec10e81 <unknown>\n#4 0x55b24ec578c4 <unknown>\n#5 0x55b24ec35b4d <unknown>\n#6 0x55b24ec54d7d <unknown>\n#7 0x55b24ec358c3 <unknown>\n#8 0x55b24ec036b3 <unknown>\n#9 0x55b24ec0468e <unknown>\n#10 0x55b24eea5b3b <unknown>\n#11 0x55b24eea9ac1 <unknown>\n#12 0x55b24ee92335 <unknown>\n#13 0x55b24eeaa642 <unknown>\n#14 0x55b24ee7749f <unknown>\n#15 0x55b24eeca038 <unknown>\n#16 0x55b24eeca203 <unknown>\n#17 0x55b24eed9f8c <unknown>\n#18 0x7f789f694ac3 <unknown>",
        "time" : {
          "start" : 1728292899016,
          "stop" : 1728292950354,
          "duration" : 51338
        }
      }, {
        "uid" : "3c1b46ee3a48afba",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/3c1b46ee3a48afba",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('css selector', 'h3')\nStacktrace:\n#0 0x55774e37c13a <unknown>\n#1 0x55774e0625e0 <unknown>\n#2 0x55774e0b1be8 <unknown>\n#3 0x55774e0b1e81 <unknown>\n#4 0x55774e0f88c4 <unknown>\n#5 0x55774e0d6b4d <unknown>\n#6 0x55774e0f5d7d <unknown>\n#7 0x55774e0d68c3 <unknown>\n#8 0x55774e0a46b3 <unknown>\n#9 0x55774e0a568e <unknown>\n#10 0x55774e346b3b <unknown>\n#11 0x55774e34aac1 <unknown>\n#12 0x55774e333335 <unknown>\n#13 0x55774e34b642 <unknown>\n#14 0x55774e31849f <unknown>\n#15 0x55774e36b038 <unknown>\n#16 0x55774e36b203 <unknown>\n#17 0x55774e37af8c <unknown>\n#18 0x7f3011e94ac3 <unknown>",
        "time" : {
          "start" : 1728289947112,
          "stop" : 1728289998460,
          "duration" : 51348
        }
      }, {
        "uid" : "1af056412a5969c6",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/1af056412a5969c6",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('css selector', 'h3')\nStacktrace:\n#0 0x5630a5ce213a <unknown>\n#1 0x5630a59c85e0 <unknown>\n#2 0x5630a5a17be8 <unknown>\n#3 0x5630a5a17e81 <unknown>\n#4 0x5630a5a5e8c4 <unknown>\n#5 0x5630a5a3cb4d <unknown>\n#6 0x5630a5a5bd7d <unknown>\n#7 0x5630a5a3c8c3 <unknown>\n#8 0x5630a5a0a6b3 <unknown>\n#9 0x5630a5a0b68e <unknown>\n#10 0x5630a5cacb3b <unknown>\n#11 0x5630a5cb0ac1 <unknown>\n#12 0x5630a5c99335 <unknown>\n#13 0x5630a5cb1642 <unknown>\n#14 0x5630a5c7e49f <unknown>\n#15 0x5630a5cd1038 <unknown>\n#16 0x5630a5cd1203 <unknown>\n#17 0x5630a5ce0f8c <unknown>\n#18 0x7fc13c894ac3 <unknown>",
        "time" : {
          "start" : 1728263351392,
          "stop" : 1728263402718,
          "duration" : 51326
        }
      }, {
        "uid" : "9aab8fef6809b886",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/9aab8fef6809b886",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('css selector', 'h3')\nStacktrace:\n#0 0x55f8704e513a <unknown>\n#1 0x55f8701cb5e0 <unknown>\n#2 0x55f87021abe8 <unknown>\n#3 0x55f87021ae81 <unknown>\n#4 0x55f8702618c4 <unknown>\n#5 0x55f87023fb4d <unknown>\n#6 0x55f87025ed7d <unknown>\n#7 0x55f87023f8c3 <unknown>\n#8 0x55f87020d6b3 <unknown>\n#9 0x55f87020e68e <unknown>\n#10 0x55f8704afb3b <unknown>\n#11 0x55f8704b3ac1 <unknown>\n#12 0x55f87049c335 <unknown>\n#13 0x55f8704b4642 <unknown>\n#14 0x55f87048149f <unknown>\n#15 0x55f8704d4038 <unknown>\n#16 0x55f8704d4203 <unknown>\n#17 0x55f8704e3f8c <unknown>\n#18 0x7f69fd894ac3 <unknown>",
        "time" : {
          "start" : 1728226498532,
          "stop" : 1728226549883,
          "duration" : 51351
        }
      }, {
        "uid" : "6c07622f38a4794a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/6c07622f38a4794a",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('css selector', 'h3')\nStacktrace:\n#0 0x55f0e3bcc13a <unknown>\n#1 0x55f0e38b25e0 <unknown>\n#2 0x55f0e3901be8 <unknown>\n#3 0x55f0e3901e81 <unknown>\n#4 0x55f0e39488c4 <unknown>\n#5 0x55f0e3926b4d <unknown>\n#6 0x55f0e3945d7d <unknown>\n#7 0x55f0e39268c3 <unknown>\n#8 0x55f0e38f46b3 <unknown>\n#9 0x55f0e38f568e <unknown>\n#10 0x55f0e3b96b3b <unknown>\n#11 0x55f0e3b9aac1 <unknown>\n#12 0x55f0e3b83335 <unknown>\n#13 0x55f0e3b9b642 <unknown>\n#14 0x55f0e3b6849f <unknown>\n#15 0x55f0e3bbb038 <unknown>\n#16 0x55f0e3bbb203 <unknown>\n#17 0x55f0e3bcaf8c <unknown>\n#18 0x7fb0d9694ac3 <unknown>",
        "time" : {
          "start" : 1728221612284,
          "stop" : 1728221663641,
          "duration" : 51357
        }
      }, {
        "uid" : "af96670c292ab57c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/af96670c292ab57c",
        "status" : "passed",
        "time" : {
          "start" : 1728176938848,
          "stop" : 1728176951372,
          "duration" : 12524
        }
      }, {
        "uid" : "1ea671132a0010e7",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/1ea671132a0010e7",
        "status" : "passed",
        "time" : {
          "start" : 1728117574363,
          "stop" : 1728117587349,
          "duration" : 12986
        }
      }, {
        "uid" : "f7e1bf6a485d4e45",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/f7e1bf6a485d4e45",
        "status" : "passed",
        "time" : {
          "start" : 1728108957478,
          "stop" : 1728108969091,
          "duration" : 11613
        }
      }, {
        "uid" : "ab4d217f69c8ab1d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/143//#testresult/ab4d217f69c8ab1d",
        "status" : "passed",
        "time" : {
          "start" : 1728090215168,
          "stop" : 1728090226603,
          "duration" : 11435
        }
      }, {
        "uid" : "5cf6b06157eb4e",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/142//#testresult/5cf6b06157eb4e",
        "status" : "passed",
        "time" : {
          "start" : 1728003829283,
          "stop" : 1728003842450,
          "duration" : 13167
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "6ea56f9ca8880d45.json",
  "parameterValues" : [ ]
}